Assets/Scripts/ScriptingSystem/FunctionBlocks/DebugN.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DebugN : FunctionBlock
+{
+    [SerializeField] private int number = 0;
+
+    private void Awake()
+    {
+        Debug.Log("awake");
+    }
+
+    public override void Act()
+    {
+        Debug.Log(number);
+    }
+}
+
```

Comment:
Is this actually used? Looks like a very simple debug class. 

If not used please remove.